### PR TITLE
Add dynamic elevation offsets to session cell

### PR DESCRIPTION
**Pull request by takahirom**
## Issue
- Add dynamic elevation offsets to session cell 
## Overview (Required)
- If you like this pull request, Please merge this(This is not necessary to merge.)
- This provide material design Dynamic elevation offsets
https://material.io/guidelines/material-design/elevation-shadows.html#
- I adjust touch event (Change to a way closer to the native RecyclerView)

## Screenshot

output

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

**Comment by konifar** [comment]
Looks cool! 🆒

File: app/src/main/res/animator/raise.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+            android:state_enabled="true"
+            android:state_pressed="true"
+            >
+        <objectAnimator
+                android:duration="@android:integer/config_shortAnimTime"
+                android:propertyName="translationZ"
```
Comment (konifar) [inline app/src/main/res/animator/raise.xml]: I have a question 🙋 
In the case of pre-lollipop device, does this work well?

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
     <!-- Elevation -->
     <dimen name="elevation_2dp">2dp</dimen>
     <dimen name="elevation_4dp">4dp</dimen>
+    <dimen name="elevation_touch_raise">6dp</dimen>
```
Comment (konifar) [inline app/src/main/res/values/dimens.xml]: Can you change name to `elevation_6dp`?

File: app/src/main/res/animator/raise.xml (hunk 1 above)
Comment (takahirom) [inline app/src/main/res/animator/raise.xml]: It does not work.
android:stateListAnimator is ignored on pre-lollipop devices.
Probably you know. I think that it is difficult to equalize .
But pre lollipop selector is working.

Comment (konifar) [inline app/src/main/res/animator/raise.xml]: Got it! Thanks!

**Comment by konifar** [comment]
I tested on my local device, it works good!

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (konifar) [inline app/src/main/res/values/dimens.xml]: What do you think about this?

Comment (takahirom) [inline app/src/main/res/values/dimens.xml]: NP.
But always touch_raise is always 6dp. So I was slightly troubled.
https://material.io/guidelines/material-design/elevation-shadows.html#elevation-shadows-elevation-android

**Comment by konifar** [comment]
LGTM! Thanks for contribution!!

**Comment by takahirom** [comment]
So sorry I forgot change resource name!!

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java [hunk 3]
```diff
@@ -151,19 +151,7 @@ private void initView() {
...
-
-            if (!isScrolling && binding.recyclerView.getScrollState() == RecyclerView.SCROLL_STATE_IDLE
-                    && e.getDownTime() > e.getEventTime() - ViewConfiguration.getTapTimeout() && e.getAction() == MotionEvent.ACTION_UP) {
-                MotionEvent down = MotionEvent.obtain(e);
-                down.setAction(MotionEvent.ACTION_DOWN);
-                binding.recyclerView.forceToDispatchTouchEvent(down);
-                binding.recyclerView.forceToDispatchTouchEvent(e);
-            }
-
+            binding.recyclerView.forceToDispatchTouchEvent(e);
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java]: Marvelous! 😇

**Pull request merged**